README.md

```diff
@@ -12,6 +12,15 @@ See also:
 ## [License](https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/LICENSE)
 Code licensed under the [MIT License](https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/LICENSE).
 
+## Branch structure
+Package Support Framework adopts a development and master branching style.
+
+### Master
+This branch is what our NuGet repository uses.  The code in this branch is the most stable.  Please do not form off this branch for development.
+
+### Develop
+This branch has the latest code, but is unstable.  Please form off this branch to make any contributions to Package Support Framework.
```

Comment:
Probably not the most accurate representation of develop. This should be the latest iteration of PSF that has features that may not exist in the NuGet package.

nit:form->fork